### PR TITLE
Flow logo

### PR DESCRIPTION
**Pull request by bwaidelich**
TASK: Adjust flow landing page

* New Logo (replaced "NEOS" by "FLOW")
* Replace licensing information in footer
* Replace "Neos" by "Flow" in footer
* Adjust contributor counter in footer

### CODE REVIEW

**Comment by bwaidelich** [comment]
image
image

File: Packages/Sites/Neos.FlowNeosIo/Resources/Private/Templates/Page/Partials/BrandLogoOutline.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<svg itemprop="logo"
+     class="
+	 	u-brandLogo
+		u-brandLogo--outline
+		{f:if(condition: '{size}', then: ' u-brandLogo--{size}')}
```
Comment (robertlemke) [inline Packages/Sites/Neos.FlowNeosIo/Resources/Private/Templates/Page/Partials/BrandLogoOutline.html]: LOL!

**Comment by robertlemke** [comment]
👍 looks good by reading, can't test it right now though

File: Packages/Sites/Neos.FlowNeosIo/Resources/Private/Templates/Page/Partials/Favicon.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<link rel="apple-touch-icon" sizes="144x144" href="{f:uri.resource(path: 'Images/Favicon/apple-touch-icon-144x144.png', package: 'Neos.NeosIo')}">
+<link rel="apple-touch-icon" sizes="152x152" href="{f:uri.resource(path: 'Images/Favicon/apple-touch-icon-152x152.png', package: 'Neos.NeosIo')}">
+<link rel="apple-touch-icon" sizes="180x180" href="{f:uri.resource(path: 'Images/Favicon/apple-touch-icon-180x180.png', package: 'Neos.NeosIo')}">
+<link rel="icon" type="image/png" href="{f:uri.resource(path: 'Images/Favicon/favicon-16x16.png', package: 'Neos.NeosIo')}" sizes="16x16">
+<link rel="icon" type="image/png" href="{f:uri.resource(path: 'Images/Favicon/favicon-32x32.png', package: 'Neos.NeosIo')}" sizes="32x32">
+<link rel="icon" type="image/png" href="{f:uri.resource(path: 'Images/Favicon/favicon-96x96.png', package: 'Neos.NeosIo')}" sizes="96x96">
+<link rel="icon" type="image/png" href="{f:uri.resource(path: 'Images/Favicon/android-chrome-192x192.png', package: 'Neos.NeosIo')}" sizes="192x192">
+<link rel="manifest" href="{f:uri.resource(path: 'Images/Favicon/manifest.json', package: 'Neos.NeosIo')}">
```
Comment (kdambekalns) [inline Packages/Sites/Neos.FlowNeosIo/Resources/Private/Templates/Page/Partials/Favicon.html]: Huh? `Images/Favicon/manifest.json` – I can haz JSON in my Image? Whatever…

**Comment by johannessteu** [comment]
Good by reading and locally testing 👍

File: Packages/Sites/Neos.FlowNeosIo/Resources/Private/Templates/Page/Partials/BrandLogoOutline.html (hunk 1 above)
Comment (bwaidelich) [inline Packages/Sites/Neos.FlowNeosIo/Resources/Private/Templates/Page/Partials/BrandLogoOutline.html]: That's just taken 1:1 from the Neos.NeosIo Package ;)

File: Packages/Sites/Neos.FlowNeosIo/Resources/Private/Templates/Page/Partials/Favicon.html (hunk 2 above)
Comment (bwaidelich) [inline Packages/Sites/Neos.FlowNeosIo/Resources/Private/Templates/Page/Partials/Favicon.html]: As well, copied from Neos.NeosIo

**Pull request merged**